dropwizard-async-http-client/src/main/java/smartthings/dw/asynchttpclient/AsyncHttpClientModule.java

```diff
@@ -0,0 +1,36 @@
+package smartthings.dw.asynchttpclient;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.google.inject.Singleton;
+import com.google.inject.multibindings.Multibinder;
+import com.google.inject.name.Named;
+import org.asynchttpclient.AsyncHttpClient;
+import org.asynchttpclient.DefaultAsyncHttpClient;
+import org.asynchttpclient.DefaultAsyncHttpClientConfig;
+import org.asynchttpclient.config.AsyncHttpClientConfigDefaults;
+import org.asynchttpclient.config.AsyncHttpClientConfigHelper;
+import org.asynchttpclient.filter.RequestFilter;
+import java.util.Set;
+
+public class AsyncHttpClientModule extends AbstractModule {
```

Comment:
Should we require `AsyncHttpClientConfig config` to be passed into a constructor of this class and then bind it to a singleton?  That's when I've been doing for the newer modules to avoid messing up the setup accidentally.  As is the config isn't bound so it would be really easy for a user to forget to do it.